docs/reports/requirements/6_test-strat.md

```diff
@@ -1,5 +1,10 @@
 # Test Strategy
 
-Section 6 of the Requirements Report
+## Test Strategy - GitHub Actions
```

Comment:
Let me know if you'd like help setting this up - we should work together on this :)